### PR TITLE
Added 'and' operator

### PR DESCRIPTION
**Pull request by Teyler7**
Allowed functionality for to-mils.js to handle operations such as "1 day and 2 mins" by returning a reduced array of the total mils.

### CODE REVIEW

File: src/to-mils.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
-    if (letters == "day" || letters == "days"){
-        return numbers * 86400000
-    } else if (letters == "hr" || letters == "hrs" || letters == "hour" || letters == "hours"){
-        return numbers * 3600000
-    } else if (letters == "min" || letters == "minute" || letters == "minutes") {
-        return numbers * 60000
-    } else if (letters == "sec" || letters == "second" || letters == "seconds"){
-        return numbers * 1000
+        if (letters == "day" || letters == "days") {
+            words[i] = numbers * 86400000
```
Comment (matthewnitschke) [inline src/to-mils.js]: replacing the words index with a number feels weird to me, can you instead keep track of a sum and just add to it? This also makes it so we don't have to loop the words twice

Something like 
```js
let sum = 0;
for(...){
    if(...){
        sum += whatever
    }
}
```

File: src/to-mils.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
 module.exports = (input) => {
-    let numbers = parseFloat(/[\d\.]+/gm.exec(input)[0])
-    let letters = (/[A-z ]+/gm.exec(input)[0]).replace(/[ ]/gm, '')
+    let words = input.split(" and ");
```
Comment (matthewnitschke) [inline src/to-mils.js]: what do you think about support for the `&` symbol

Comment (matthewnitschke) [reply src/to-mils.js]: Genuinely a question about it, not saying we should implement it

File: test/test.js [hunk 3]
```diff
@@ -62,4 +62,28 @@ describe('Tests', () => {
 		expect(toMs('.62 sec')).to.equal(msInSec * 0.62, '.62 sec')
 		expect(toMs('0.54 sec')).to.equal(msInSec * 0.54, '0.54 sec')
 	})
+
+	it('should test and operator functionality', () => {
+		expect(toMs('1 day and 1 day')).to.equal((msInDay + msInDay), '1 day and 1 day')
```
Comment (matthewnitschke) [inline test/test.js]: Can you test for situations where the value isn't just `1`? Also test for decimal value usages?

Comment (Teyler7) [reply test/test.js]: You betcha, skipper

**Pull request merged**